### PR TITLE
[stable/prometheus] add node-exporter podsecuritypolicy support

### PR DESCRIPTION
**Pull request by spiarh**
Signed-off-by: Ludovic Cavajani <lcavajani@suse.com>

#### What this PR does / why we need it:
This PR allows one to create a specific podSecurityPolicy for node-exporter. This is very useful in an environment where podSecurityPolicy is enabled. 

The podSecurityPolicy allows node-exporter to use:
* hostPath (/proc, /sys) by default + extraHostPathMount if any
* hostNetwork if enabled
* hostPID if enabled

Otherwise, it adds extra security.

For more info on podSecurityPolicy, here is the [link](https://kubernetes.io/docs/concepts/policy/pod-security-policy) to the official documentation.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/prometheus/templates/node-exporter-podsecuritypolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{{- if .Values.nodeExporter.podSecurityPolicy.enabled }}
```
Comment (gianrubio) [inline stable/prometheus/templates/node-exporter-podsecuritypolicy.yaml:2]: I think you also need to check if `rbac.create` is true

File: stable/prometheus/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: prometheus
-version: 7.2.0
+version: 7.2.1
```
Comment (gianrubio) [inline stable/prometheus/Chart.yaml]: please bump to a major version

**Comment by lcavajani** [comment]
I made the changes :)

**Comment by lcavajani** [comment]
I found there is an [issue (8126)](https://github.com/helm/charts/issues/8126) regarding this.

**Pull request merged**